docs/content/ingestion/data-formats.md

```diff
@@ -87,12 +85,19 @@ Since the CSV data cannot contain the column names (no header is allowed), these
   }
 ```
 
-The `columns` field must match the columns of your input data in the same order.
+#### CSV Index Tasks
```

Comment:
People aren't going to understand what this means. I'd suggest being much more explicit. A sentence like this should appear:

The `hasHeaderRow` parameter is only supported for tasks of type "index", and is ignored for any other type, including any realtime or Hadoop-based tasks.